TypeScript error with iron-form-element-behavior 2.1.2

[This recent update](https://github.com/PolymerElements/iron-form-element-behavior/commit/664f99e825a3ee877aa6644283bfe8a8708dcde5) in `iron-form-element-behavior` breaks the TypeScript compilation:

The `value` property was changed from …

```
value: string|null|undefined;
```
… to …
```
value: string|number|null|undefined;
```

Which produces the following error:

```
error TS2320: Interface 'PaperInputElement' cannot simultaneously extend types 'PaperInputBehavior' and 'IronFormElementBehavior'.
  Named property 'value' of types 'PaperInputBehavior' and 'IronFormElementBehavior' are not identical.
```

Fix should be as easy as updating the corresponding definition in `paper-input` accordingly. However, I'm not sure whether the dependency version here should be adjusted as well? Strictly speaking, this looks like a breaking change from a TS perspective to me.

--- a/paper-input.d.ts
+++ b/paper-input.d.ts
@@ -67,6 +67,7 @@
  * `--paper-input-container-ms-clear` | Mixin applied to the Internet Explorer reveal button (the eyeball) | {}
  */
 interface PaperInputElement extends Polymer.Element, Polymer.PaperInputBehavior, Polymer.IronFormElementBehavior {
+  value: string|null|undefined;
 
   /**
    * Returns a reference to the focusable element. Overridden from PaperInputBehavior

--- a/paper-input.html
+++ b/paper-input.html
@@ -276,6 +276,13 @@ Custom property | Description | Default
       Polymer.IronFormElementBehavior
     ],
 
+    properties: {
+      value: {
+        // Required for the correct TypeScript type-generation
+        type: String
+      }
+    },
+
     beforeRegister: function() {
       // We need to tell which kind of of template to stamp based on
       // what kind of `iron-input` we got, but because of polyfills and

